src/RepositoryIdAwareTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle;
+
+/**
+ * Trait RepositoryIdAwareTrait.
+ *
+ * @internal For use in EzSystems\PlatformHttpCacheBundle package.
+ */
+trait RepositoryIdAwareTrait
```

Comment:
IMHO we can introduce small utility service instead of using Traits here: 

```php
/**
 * @internal For use in EzSystems\PlatformHttpCacheBundle package.
 */
final class RepositoryPrefixProvider 
{	
	public function getRepositoryPrefix(string $repositoryId, array $repositories): string
	{
		// ...
		return empty($repositoryId) || $repositoryId === $default ? '' : $repositoryId . '_';
	}
}
```

Reply:
Indeed, I'll look into that when I better understand why the one REST test fails here.

Reply:
Pushed change for this, but opted to use config resolver as it might actually solve the failure _(if the failure is caused by scope change issues...)_.